docs/user-guide/manage-environments.md

```diff
@@ -178,7 +178,25 @@ To inspect the deployed application in KubeRocketCI portal, take the following s
 
     ![Inspect application](../assets/user-guide/environments/application_terminal.png "Inspect application")
 
+### Delete Application
+
+There are two buttons in the environment which can be used in the portal:
+
+  ![Clean button](../assets/operator-guide/edp-portal-clean-application.png "Clean button")
+
+  * **Delete** - Deletes selected applications and associated resources.
```

Comment:
Manages custom cleanup actions, such as deleting cloud resources and databases, rolling back transactions, etc.